pkg/proxy/handlers/spacelister_test.go

```diff
@@ -74,6 +81,8 @@ func buildSpaceListerFakes(t *testing.T) (*fake.SignupService, *test.FakeClient)
 		fake.NewSpace("grandchildspace", "member-1", "grandchildspace", spacetest.WithSpecParentSpace("childspace")),
 		// noise space, user will have a different role here , just to make sure this is not returned anywhere in the tests
 		fake.NewSpace("otherspace", "member-1", "otherspace", spacetest.WithSpecParentSpace("otherspace")),
+		// space flagged as community
+		fake.NewSpace("communityspace", "member-2", "communityspace"),
```

Comment:
should we append this space only if `publicViewerEnabled` ? I mean do it as you do for the spacebindings at line 107 ?

Reply:
ditto